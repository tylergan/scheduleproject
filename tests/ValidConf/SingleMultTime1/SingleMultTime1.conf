on Saturday,Tuesday at 1654,1200 run /bin/echo hello