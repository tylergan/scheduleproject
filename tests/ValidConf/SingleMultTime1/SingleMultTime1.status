will run at XX XX XX 12:00:00 2020 /bin/echo hello
will run at XX XX XX 16:54:00 2020 /bin/echo hello
will run at XX XX XX 12:00:00 2020 /bin/echo hello
will run at XX XX XX 16:54:00 2020 /bin/echo hello
ran XX XX XX 12:00:00 2020 /bin/echo hello
will run at XX XX XX 16:54:00 2020 /bin/echo hello
will run at XX XX XX 12:00:00 2020 /bin/echo hello
will run at XX XX XX 16:54:00 2020 /bin/echo hello
ran XX XX XX 12:00:00 2020 /bin/echo hello
ran XX XX XX 16:54:00 2020 /bin/echo hello
will run at XX XX XX 12:00:00 2020 /bin/echo hello
will run at XX XX XX 16:54:00 2020 /bin/echo hello
ran XX XX XX 12:00:00 2020 /bin/echo hello
ran XX XX XX 16:54:00 2020 /bin/echo hello
ran XX XX XX 12:00:00 2020 /bin/echo hello
will run at XX XX XX 16:54:00 2020 /bin/echo hello
ran XX XX XX 12:00:00 2020 /bin/echo hello
ran XX XX XX 16:54:00 2020 /bin/echo hello
ran XX XX XX 12:00:00 2020 /bin/echo hello
ran XX XX XX 16:54:00 2020 /bin/echo hello
