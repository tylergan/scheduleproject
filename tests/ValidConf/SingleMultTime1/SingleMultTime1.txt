Testing that the output for runner.py and runner.status works when: 
    1) Running a single line specifying multiple times (testing two days and times)