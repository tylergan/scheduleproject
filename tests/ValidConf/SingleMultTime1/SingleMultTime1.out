hello
hello
hello
hello
nothing left to run
